### ISSUE TITLE
Модификация каскадом

### ISSUE THREAD

**Issue by mustakissa** [bem/bem-forum-content-ru#884]
Можно ли каскадом от модификатора изменять вложенные сущности?

Предположим, есть три одинаковые колонки, но элементы в средней колонке отличаются от элементов в крайних. Я могу каждому из этих элементов дать свой модификатор и получить то, что нужно:

https://jsfiddle.net/h9ms6moy/

А могу задать один модификатор самой колонке, и каскадом от него получить такой же результат:

https://jsfiddle.net/h9ms6moy/1/

Я где-то слышал, что последний вариант приемлем, правда, речь шла о каскаде от модификатора не элемента, а блока. 

Противоречят ли такие приемы методологии?

Прошу обратить внимание, что в конкретном случае модификатор сам по себе не несет никаких стилей, а используется только для каскада.

**Comment by tadatuta** [comment]
@mustakissa Да, вложенные селекторы здесь вполне приемлемы. См. [FAQ](https://ru.bem.info/faq/#Почему-нежелательно-использовать-вложенные-селекторы).

**Comment by mustakissa** [comment]
@tadatuta Спасибо за ответ.
